apis/otel/v1alpha1/splunkotelagent_webhook.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	defaultJavaAgentImage = "quay.io/signalfx/splunk-otel-instrumentation-java:v1.6.0"
+	defaultJavaAgentImage = "quay.io/signalfx/splunk-otel-instrumentation-java:latest"
```

Comment:
Why do you think it should be latest? Maybe stick with v1 to avoid potential breaking changes in v2?

Reply:
Good point. Java team does not publish a v1 tag. I'll create a PR for that and update this repo after.